packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -697,6 +793,60 @@ class MemoryController extends DisposableController
     });
   }
 
+  AnalysesReference findAnalysesNode() {
+    for (final child in topNode.children) {
+      if (child is AnalysesReference) {
+        return child;
+      }
+    }
+    return null;
+  }
+
+  final realNodeFromFakeNodeNotifier = ValueNotifier<Reference>(null);
```

Comment:
this doesn't seem like it is used. If it is used, give it a clearer name.

Reply:
It's dead code, removed.